### PR TITLE
Follow System Pagination added

### PR DESCRIPTION
**Pull request by cakmadam98**
Unit tests are passed since it is backwards compatible. Also I've corrected a small typo in the FollowRequest documentation.

### CODE REVIEW

File: Platon/backend/app/follow_system/forms.py [hunk 1]
```diff
@@ -6,31 +6,43 @@
...
     follower_id = IntegerField('follower_id', validators=[validators.DataRequired()])
+    page = IntegerField("page")
+    per_page = IntegerField("per_page")
 
 get_followings_parser = reqparse.RequestParser()
 get_followings_parser.add_argument('follower_id', required=True, type=int,
                                    help="ID of the follower. Follower is the one who follows someone.",
                                    location='form')
+get_followings_parser.add_argument('page',type=int,help="Page index that you want(Starts from 0)",location='form')
+get_followings_parser.add_argument('per_page',type=int,help="Number of items in a page",location='form')
```
Comment (HUmutOzdemir) [inline Platon/backend/app/follow_system/forms.py:17]: Why did you took this parameters form the body? As I know there is no body content in GET/HEAD requests.

File: Platon/backend/app/follow_system/forms.py [hunk 2]
```diff
@@ -6,31 +6,43 @@
...
     following_id = IntegerField('following_id', validators=[validators.DataRequired()])
+    page = IntegerField("page")
+    per_page = IntegerField("per_page")
 
 get_followers_parser = reqparse.RequestParser()
 get_followers_parser.add_argument('following_id', required=True, type=int,
                                   help="ID of the following user. Following user is the one who is followed by someone.",
                                   location='form')
+get_followers_parser.add_argument('page',type=int,help="Page index that you want(Starts from 0)",location='form')
+get_followers_parser.add_argument('per_page',type=int,help="Number of items in a page",location='form')
```
Comment (HUmutOzdemir) [inline Platon/backend/app/follow_system/forms.py:31]: Same as above

File: Platon/backend/app/follow_system/forms.py [hunk 3]
```diff
@@ -6,31 +6,43 @@
...
     following_id = IntegerField('following_id', validators=[validators.DataRequired()])
+    page = IntegerField("page")
+    per_page = IntegerField("per_page")
 
 get_follow_requests_parser = reqparse.RequestParser()
 get_follow_requests_parser.add_argument('following_id', required=True, type=int,
                                         help="ID of the following user. Following user is the one who is followed by someone.",
                                         location='form')
+get_follow_requests_parser.add_argument('page',type=int,help="Page index that you want(Starts from 0)",location='form')
+get_follow_requests_parser.add_argument('per_page',type=int,help="Number of items in a page",location='form')
```
Comment (HUmutOzdemir) [inline Platon/backend/app/follow_system/forms.py:45]: Same as above

**Review by HUmutOzdemir — CHANGES_REQUESTED** [review]

**Review by HUmutOzdemir — APPROVED** [review]
After the last changes that you did. I think we can merge your code.

**Review by hilal-demir — APPROVED** [review]

**Pull request merged**